packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -47,13 +47,17 @@ const propTypes = {
    * The intl object to be injected for translations.
    */
   intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
+  inputAttrs: PropTypes.object,
```

Comment:
this prop is missing prop usage information. `inputAttrs` does not sound relative to button 
should be renamed like `buttonAttrs` OR equivalent text.